library/network_connections.py

```diff
@@ -1629,6 +1586,21 @@ def fail_json(
 ###############################################################################
 
 
+class NmLsrLogHanlder(logging.Handler):
```

Comment:
There is a typo. Also, why `Lsr` here? This is all about the linux system roles, so "Lsr" is implied by the context.